package.json

```diff
@@ -114,9 +113,8 @@
   },
   "peerDependencies": {
     "moment": "^2.18.1",
-    "react": ">=0.14",
-    "react-dom": ">=0.14",
-    "react-addons-shallow-compare": ">=0.14"
+    "react": ">=15.3.0",
+    "react-dom": ">=15.3.0"
```

Comment:
Hey @ljharb, I feel like it's a reasonable thing for us to expect `15.3.0` or higher at this point in time, but I'm unsure... do you think this is fine?